File: CodeWars/JavaScript/Build Tower.js
// https://www.codewars.com/kata/576757b1df89ecf5bd00073b/train/javascript
function towerBuilder(nFloors) {
    const floorLength = 1+2*(nFloors-1)
    const tower = []
    let i = 1
    while (tower.length < nFloors) {
        let space = floorLength - i
        let floor = ""
        if (space>0) {floor = floor.concat(' '.repeat(space/2))}
        floor = floor.concat('*'.repeat(i))
        if (space>0) {floor = floor.concat(' '.repeat(space/2))}
        tower.push(floor)
        i+=2
        
    }
    return tower
}

const arr = towerBuilder(2)
console.log(arr)

File: CodeWars/Python/build_tower.py
## https://www.codewars.com/kata/576757b1df89ecf5bd00073b/train/python
def tower_builder(n_floors):
    tower = []
    floor_length = 1 + 2*(n_floors-1)
    i = 1
    while len(tower) < n_floors:
        space = (floor_length - i)/2
        floor = ""
        if space > 0:
            for x in range(int(space)):
                floor += ' '
        for x in range(i):
            floor += "*"
        if space > 0:
            for x in range(int(space)):
                floor += ' '
        tower.append(floor)
        i+=2
    return tower

arr = tower_builder(3)
print(arr)